engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -72,6 +73,7 @@
     // cached/transient state
     private transient DataCleanerConfiguration _dataCleanerConfiguration;
     private transient AnalysisJobBuilder _analysisJobBuilder;
+    private AnalysisJobBuilder _analysisJobBuilder1;
```

Comment:
It will... It even warns when committing, but it often drowns in other warnings from to silly coding practices and bad IDEs :p